### PR TITLE
webUI updates

### LINKED ISSUE #1706: Update WebUI
## Problem Definition

We need to adapt the WebUi to work with the changed API.

Ref #1669

### PR DESCRIPTION
**Pull request by andrevmatos**
Some API changes were needed in order to get a cohesive API that could be used by the web client:
- `GET /channels` and `GET /channels/<token_address>/<partner_address>` both are encoded with the same serializer, `ChannelStateSchema`, to provide the same channel information layout (the former, a list of channel objects, the later a single channel object)
- `GET /events/channels/<token_network_address>/<channel_identifier> -> /events/channels/<token_address>/<channel_identifier>`: this endpoint was the only one using token network address, which isn't immediately available to clients, so channel specific events couldn't be fetched. Now, it uses `token_address`, as the other APIs
Fix #1706

### CODE REVIEW

**Comment by LefterisJP** [comment]
@andrevmatos so far looking okay, can you also add [ci integration] when done so that all API tests are also ran in travis to make sure the changes don't break anything?

**Comment by andrevmatos** [comment]
Yes, I think so, just didn't do it here to minimize the changes, but I think it's good to keep consistency on this

**Comment by LefterisJP** [comment]
> Do you think we should convert it to events/channels/<token_address>/<partner_address> instead to be consistent with the other endpoints?

Both should be available. Can you do this @andrevmatos ?

- `events/channels/<token_address>/` should return all channel events for `token_address`

- `events/channels/<token_address>/<partner_address> ` should filter it further and return all channel events for `token_address` but only for `partner_address`

File: raiden/api/python.py [hunk 1]
```diff
@@ -653,6 +658,7 @@ def get_channel_events(
                 new_event.update(event.__dict__)
                 returned_events.append(new_event)
 
+        returned_events.sort(key=lambda evt: evt.get('block_number'))
```
Comment (LefterisJP) [inline raiden/api/python.py]: Nice. But for consistency please make sure that all events endpoints follow this behaviour.

File: raiden/api/v1/resources.py [hunk 2]
```diff
@@ -98,8 +98,6 @@ class NetworkEventsResource(BaseResource):
 
     @use_kwargs(get_schema, locations=('query',))
     def get(self, from_block, to_block):
-        from_block = from_block or 0
```
Comment (LefterisJP) [inline raiden/api/v1/resources.py]: Leave those here please. I am changing those in my next PR.

File: raiden/api/v1/resources.py [hunk 3]
```diff
@@ -113,8 +111,6 @@ class TokenEventsResource(BaseResource):
 
     @use_kwargs(get_schema, locations=('query',))
     def get(self, token_address, from_block, to_block):
-        from_block = from_block or 0
```
Comment (LefterisJP) [inline raiden/api/v1/resources.py]: leave them as they were

File: raiden/api/v1/resources.py [hunk 4]
```diff
@@ -127,12 +123,10 @@ class ChannelEventsResource(BaseResource):
     get_schema = EventRequestSchema()
 
     @use_kwargs(get_schema, locations=('query',))
-    def get(self, token_address, channel_identifier, from_block, to_block):
-        from_block = from_block or 0
```
Comment (LefterisJP) [inline raiden/api/v1/resources.py]: same here, leave them as they were

**Comment by LefterisJP** [comment]
Merging this. The failing tests are the ones that we are working to fix on https://github.com/raiden-network/raiden/issues/1733

**Pull request merged**